newsletter-february-2023.rst

```diff
@@ -0,0 +1,119 @@
+.. post:: Feb 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - February 2023
+========================================
+
+News and updates
+----------------
+
+Here are the latest updates from our team since the :doc:`previous newsletter </newsletter-january-2023>`:
+
+- 🪄️ Build outputs are now stored in a well-known location, available to the build process.
+  This opens up many new and exciting possibilities for generating and processing final output formats,
+  which we will uncover in an upcoming blog post.
+  PDFs for MkDocs and encrypted documentations are just two demos that we have ready.
+  **Stay tuned!**
+- 🌪️ Uploads are faster and done in parallel,
+  thanks to `rclone`_. 
+- 🔒️ CORS headers have been tightened on all platforms.
+- 🛠️ The default branch of a git repository is now correctly detected in manual imports.
+  This was a long-standing bug that we are very happy to finally have fixed.
+- 🎤️ Write the Docs has announced their 2023 conferences.
+  Read more in `their announcement`_.
+- 🔒️ Security issue found and fixed: `Path traversal: access to files from any project <GHSA-5w8m-r7jm-mhp9>`_
+- 🔒️ Security issue found and fixed: `Symlink following: Arbitrary file access from builder server <GHSA-hqwg-gjqw-h5wg>`_
+- 🔒️ Security issue found and fixed: `Cache poisoning <GHSA-7fcx-wwr3-99jv>`_
+- 🚦️ Note from Cloudflare that provides CDN for everything on readthedocs.io:
```

Comment:
```suggestion
- 🚦️ A maintenance update from Cloudflare that provides our CDN and SSL provisioning. 
We are working to ensure that nothing is offline other than creation of new domains:
```

Reply:
Also seems like this should be in the Issues section?

Reply:
I agree, will move it. It also makes the issues section more relevant to look at.